src/newspack-ui/scss/elements/woocommerce/_overrides.scss

```
@@ -21,18 +21,3 @@
 		}
 	}
 }
```

```suggestion
}

.woocommerce-account {
	// Inline sign up form max-width.
	.woocommerce-notices-wrapper:has(+ .newspack-reader-auth__inline-wrapper),
	.newspack-reader-auth__inline-wrapper {
		margin-left: auto;
		margin-right: auto;
		max-width: var(--newspack-ui-modal-width-s);
	}

	.newspack-reader-auth__inline-wrapper h2 {
		font-size: var(--newspack-ui-font-size-m);
		font-weight: 600;
	}
}
```

This all should stay -- I double-checked on this branch, and we're still using the hard-coded form (and not the block) on the /my-account page; this CSS just makes sure it looks a little less blown out and weird (see https://github.com/Automattic/newspack-plugin/pull/3292)